### DISCUSSION TITLE
.github repository not working?

### DISCUSSION THREAD

**Discussion by ChlodAlejandro** [community/community#41994] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I recently found out about the special `.github` repository that you can make, and I've decided to begin using this for my profile and default `FUNDING.yml` file. I used to have a repository under my username (as described [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme)), but after seeing that you can achieve the same thing by having a `profile/README.md` in this special repository, I decided to switch over. After settling on the format, I did a bit of a switcheroo which involved renaming and deleting repositories, but now the `.github` repository doesn't work at all. I used to see a sponsor link on all my repos, but that's no longer the case now. Same with the profile README. I still do have the repo under [the correct name](https://github.com/ChlodAlejandro/.github) (and was created with that name just to be sure), so I'm not sure why it's not showing up at all. Thinking this might be due to a (now-deleted) past repo named `.github`, or some caching problem. Any suggestions?

**Comment by turangarusso** [comment]
Try to delete the repo with your username, and the profile Readme, try also to delete browser cache or change it and repeat the process.

**Comment by radar-blip** [comment]
Your README.md is inside a folder, take it out of the folder and it should work.

**Comment by ChlodAlejandro** [comment, marked as answer]
Taking it out of the `profile` folder (as suggested by @linuxlawson) didn't work. I also already deleted the repo yesterday just to test out if that would work.

The sponsor button has returned to *a few* repos, so I'm thinking this is just a caching thing on GitHub's side. The `profile` README, however, has not returned, so I just recreated the username repository and re-uploaded the README there (instead of relying on a `profile` folder inside the `.github` repo). I'm guessing this is the only place the README gets checked for user accounts.

In any case, thank you for the suggestions, everyone!
